### PR TITLE
Assessments dashboard analytics

### PR DESCRIPTION
**Pull request by cforkish**
simple update to add analytics to the assessments tab of the teacher dashboard.

## Links

- jira ticket: [LP-1882]

[LP-1882]: https://codedotorg.atlassian.net/browse/LP-1882

### CODE REVIEW

File: apps/package.json [hunk 1]
```diff
@@ -253,7 +253,7 @@
     "qrcode.react": "^0.8.0",
     "query-string": "4.1.0",
     "react-ace": "^9.2.1",
-    "react-csv": "^1.0.14",
+    "react-csv": "^2.0.3",
```
Comment (jamescodeorg) [inline apps/package.json:256]: Just checking that this change was intentional?

File: apps/src/templates/sectionAssessments/SectionAssessments.jsx [hunk 2]
```diff
@@ -87,12 +88,63 @@ class SectionAssessments extends Component {
...
+    this.logEvent('select_student', {
+      student_id: studentId,
+      script_id: scriptId,
+      level_group_id: assessmentId
+    });
+  };
+
+  onClickDownload(dataType) {
+    const {assessmentId, scriptId} = this.props;
+    this.logEvent('download_results', {
```
Comment (jamescodeorg) [inline apps/src/templates/sectionAssessments/SectionAssessments.jsx]: I think putting all of the different download type in the same event might make it more difficult to query for, say, just the events related to downloading assessment data (right?).  Do we want to put the dataType as part of the event name (e.g. "download_surveys", "download_feedback", etc.) instead?

File: apps/package.json (hunk 1 above)
Comment (cforkish) [reply apps/package.json:256]: yeah, the old version didn't have an `onClick` property. i documented that in the commit message but forgot to add it to the PR description.

File: apps/src/templates/sectionAssessments/SectionAssessments.jsx (hunk 2 above)
Comment (cforkish) [reply apps/src/templates/sectionAssessments/SectionAssessments.jsx]: this is what Amanda asked for but i'll double check

Comment (cforkish) [reply apps/src/templates/sectionAssessments/SectionAssessments.jsx]: Amanda agrees, will update!

**Pull request merged**